root/static/scripts/edit/forms.js

```diff
@@ -30,7 +30,9 @@ MB.forms = {
 
         function buildOptions(parent, indent) {
             var i = 0, children = parent.children, child;
-            if (!children) { return; }
+            if (!children) { 
+              return; 
```

Comment:
Indentation here should be 4 spaces to be consistent with the rest of the file

Reply:
I have added the indentation. Sorry about that.